docs/source/using_pyumbral.rst

```diff
@@ -107,7 +107,7 @@ S3, IPFS, Google Cloud, Sneakernet, etc.
 Bob fails to open the capsule
 -------------------------------
 If Bob attempts to open a capsule that was not encrypted for his public key,
-or re-encrypted for him by Ursula, He will not be able to open it.
+or re-encrypted for him by Ursula, he will not be able to open it.
```

Comment:
I take it you didn't like the religious overtones?

Reply:
Thou shalt not take the name of the Lord thy God in vain